### ISSUE TITLE
[ACT] HTML page has lang attribute - Failed Examples (fail vs inapplicable and fail vs passed)

### ISSUE THREAD

**Issue by philljenkins** [IBMa/equal-access#1020]
Investigate rule differences between ACT fail case and Checker reporting passing and/or inapplicable for the following:

[HTML page has lang attribute](https://wai-wcag-act-rules.netlify.app/standards-guidelines/act/implementations/equal-access/#breakdown-b5c3f8)

[Failed Example 1](https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/b5c3f8/941efb7368e46b27b937d34b07fc4d41da01b002.html)
OK: ACT fail vs fail - WCAG20_Html_HasLang: Pass_0, Fail_3
ACT fail vs inapplicable - WCAG20_Elem_Lang_Valid: Pass_0, Fail_1

[Failed Example 3](https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/b5c3f8/5636ffbff7668ca38610e36905889df90a92f485.html)
ACT fail vs passed - WCAG20_Html_HasLang:Pass_0,Fail_3
OK: ACT fail vs fail - WCAG20_Elem_Lang_Valid:Pass_0,Fail_1

[Failed Example 4](https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/b5c3f8/4885abc44ca092b3d07a3f8e855dbcbf33f6c66b.html)
OK: ACT fail vs fail - WCAG20_Html_HasLang:Pass_0,Fail_3
ACT fail vs passed - WCAG20_Elem_Lang_Valid:Pass_0,Fail_1

### Make recommendation:
[ ] Either fix Checker rue, or make recommendation to fix ACT rule and/or testcase
[ ] Open issue 

### Background:
The comparison analysis is documented in a spreadsheet here:
See [Checker_ACT_Mapping 2022-06-23.xlsx](https://ibm.ent.box.com/file/981548094874)
(see column H cells highlighted in yellow for which Checker rules need changing)

Also see the **_Checker Implementation Report_** which documents the rules run against the ACT test cases.
Equal Access Accessibility Checker version 3.1.33-rc, test results were last updated 7 July 2022. 
https://wai-wcag-act-rules.netlify.app/standards-guidelines/act/implementations/equal-access/ 

![image](https://user-images.githubusercontent.com/16193609/182182670-1beada8e-44de-40f1-b97f-5b478bc39ea9.png)

**Comment by aliunwala** [comment]
Triage:

This might also not be a problem. The fail on Example 3 should take priority over the pass.

**Comment by tombrunet** [comment]
See #1022 regarding untested. Waiting on respond from Wilco.

**Comment by tombrunet** [comment]
@philljenkins Results are consistent now, but we report 3.1.1 and 3.1.2 whereas they only report 3.1.1

**Comment by philljenkins** [comment]
@tombrunet The issue to split the Checker rule into two rules to distinguish and report differently between "a page's language" and "a part's language" may close this issue.
#1018

**Comment by philljenkins** [comment]
ACT Implementation report requires that "**_Fail testcases_**" should be failed by _at least one_ of the Checker Rules (not both). The Checker rules cover all 7 ACT test cases.  

Implementation is "**_Partially consistent_**" because the rules are reporting/mapping against different WCAG Success Criteria than ACT (3.1.1 Page and 3.1.2 Parts whereas ACT only report 3.1.1 Page) as noted in issue #1018.

So closing this issue as it no longer needed.
